src/tools/prte/prte.1.md

```diff
@@ -60,30 +95,31 @@ cluster to use for the DVM.
 
 :   Synonym for `-hostfile`.
 
-`--prefix <dir>`
-
-:   Prefix directory that will be used to set the `PATH` and
-    `LD_LIBRARY_PATH` on the remote node before invoking the PRTE
-    daemon.
-
 Setting MCA parameters:
 
-`--gmca <key> <value>`
+`--gpmixmca <key> <value>`
 
-:   Pass global MCA parameters that are applicable to all contexts.
+:   Pass global PMIx MCA parameters that are applicable to all contexts.
     `<key>` is the parameter name; `<value>` is the parameter value.
 
 `--mca <key> <value>`
 
 :   Send arguments to various MCA modules. See the "MCA" section,
     below.
 
```

Comment:
There's a --pmixmcam option below this that mentions a MCA parameter file set list. Is the 'arg0' supposed to be a path to the file? I'm not sure what it is. Also, maybe the same comments I made about global PMIx MCA parameters and contexts apply here. I'm also not sure I understand why I would have global vs context-specific settings here or how I would control that since I don't see any way to specify context.

Reply:
Good point. I'll clarify.
 * The `arg0` is supposed to be a file path
 * The global MCA's apply to all app contexts in a MIMD launch (`prterun -gpmixmca foo bar ./a.out : ./b.out`) whereas the non-global MCA's (for the most part) apply only to that specific app context (`prterun -pmixmca foo bar ./a.out : -pmixmca foo baz ./b.out`)